pkg/releaselock/releaselock.go

```diff
@@ -0,0 +1,89 @@
+package releaselock
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"net"
+	"net/rpc"
+	"strconv"
+
+	"github.com/prashanthpai/sunrpc"
+	"k8s.io/klog/v2"
+)
+
+const (
+	pmapProgramNumber  = uint32(100021)
+	pmapProgramVersion = uint32(4)
+	programNumber      = uint32(200002)
+	programVersion     = uint32(1)
+	procedureNumber    = uint32(1)
+	procedureName      = "IN_BAND_PROPRIETARY_LOCK_OPS_PROG.IN_BAND_PROPRIETARY_LOCK_OPS_V1"
+	rpcPort            = "111"
+	protocol           = "tcp"
+)
+
+type releaseLockResponse struct {
+	status releaseLockStatus
+}
+
+type releaseLockStatus uint
+
+// ReleaseLock calls the NFS server to remove all advisory locks on the given GKE node.
+// hostIP is the internal IP address of the Filestore instance.
+// clientIP is the internal IP address of the GKE node.
+func ReleaseLock(hostIP, clientIP string) error {
+	procedureID := sunrpc.ProcedureID{
+		ProgramNumber:   programNumber,
+		ProgramVersion:  programVersion,
+		ProcedureNumber: procedureNumber,
+	}
+	procedure := sunrpc.Procedure{
+		ID:   procedureID,
+		Name: procedureName,
+	}
+	if err := sunrpc.RegisterProcedure(procedure, true /* validateProcName */); err != nil {
```

Comment:
is this RegisterProcedure needed for every call? I think we can do it once as part of package init.

Reply:
Saikat seem to be right, there is no need to do this every time.  Could be done once in initialization.

Reply:
Done. Remove it to a different func, which will be called during driver initialization.